### ISSUE TITLE
API support for verified artists

### ISSUE THREAD

**Issue by VirUZI** [spotify/web-api#10]
As far as I can tell Artists and Users (Profiles) are currently two separates entities in the API with no connection between them. In the Spotify Client however the concepts are merged for Verified Artists so information like playlists from the linked user account can be displayed on the artist page and there is a link from the user profile to the artist discography.

I would like to retrieve this information in the API so that I for instance can get "Artist Playlists" via an artist object or the albums for an artist via an user profile.

**Comment by mgingras** [comment]
:+1:

**Comment by patmanh** [comment]
:+1: +1

**Comment by rickyrombo** [comment]
:+1: Bump! I need a way to figure out which artist the authenticated user can manage.

**Comment by michaelherger** [comment]
Would give us access to the "Artist Playlists"

**Comment by hughrawlinson** [comment]
Nope, no plans in the near future. I'll make sure to update this ticket if/when there are :)

**Comment by aideenf** [comment]
yes this would be a great feature please 👍

**Comment by haywirez** [comment]
Would like to have this as well

**Comment by xlcrr** [comment]
+1

**Comment by dougbutner** [comment]
Is there any progress on this 

> Nope, no plans in the near future. I'll make sure to update this ticket if/when there are :)
Any progress on this feature? Please respond either way, so I can make a decision for my app cXc Music!! Thanks!

**Comment by bhavifm** [comment]
Any update on this feature ?

**Comment by MaskyS** [comment]
Is there anything we can do to speed up or at least make any progress on this? I'd like to check if users are verified artists after they login using Spotify to grant them special privileges.
